src/routes/errorPage.tsx

```diff
@@ -0,0 +1,17 @@
+import { useRouteError } from "react-router-dom";
+
+export default function ErrorPage(): JSX.Element {
+  // add return type to function
+  const error: any = useRouteError();
+  console.error(error);
```

Comment:
remove print to console statement

Reply:
And don't forget to remove the comments as well